rpcs3/Emu/RSX/Program/GLSLSnippets/VideoOutCalibrationPass.glsl

```diff
@@ -0,0 +1,145 @@
+R"(
+#version 420
+
+#ifdef VULKAN
+layout(set=0, binding=1) uniform sampler2D fs0;
```

Comment:
Add a macro above the #ifdef vulkan to calculate the sampler locations. We sometimes have to move them around.
Just add something like `#define SAMPLER_LOCATION(x) (x + %loc)` and then do a string replace for %loc in the backend. Then you can set binding to SAMPLER_LOCATION(0) and SAMPLER_LOCATION(1).
- For GL, we have GL_TEMP_IMAGE_SLOT which is currently 31
- For VK we use pipeline-layout indices which are different for each shader unlike GL. Just replace %loc with 1 and you're good to go. Slot 0 is used for UBOs, but I got lazy and forgot to define an enum for the layout.

Reply:
This won't work because it's 31 and 30 instead of 31 and 32

Reply:
Just pass the integer values in the cpp side then, I'll fix up the constant names later.